aws/table_aws_cloudfront_origin_access_identity.go

```diff
@@ -0,0 +1,163 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudfront"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCloudFrontOriginAccessIdentity(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_cloudfront_origin_access_identity",
+		Description: "AWS CloudFront Origin Access Identity",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("id"),
+			ShouldIgnoreError: isNotFoundError([]string{"NoSuchCloudFrontOriginAccessIdentity"}),
+			Hydrate:           getCloudFrontOriginAccessIdentity,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listCloudFrontOriginAccessIdentities,
+		},
+		Columns: awsRegionalColumns([]*plugin.Column{
```

Comment:
This is not a regional resource, I don’t  think we need to use `awsRegionalColumns`,  please check .